config/locales/en.yml

```diff
@@ -35,7 +35,7 @@ en:
     url_online_chat: https://beta-getintoteaching.education.gov.uk/#talk-to-us
   page_titles:
     application_form: Your application
-    application_dashboard: Application dashboard
+    application_dashboard: Your application
```

Comment:
Can we pluralise this based on the same logic as the h1 heading please?

Reply:
Ah yes, good spot, the page title needs pluralising. Added the logic to the `complete.html.erb` template.